flask_webapp/Pipfile

```diff
@@ -0,0 +1,37 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+aniso8601 = "==8.0.0"
+bluepy = "==1.3.0"
+certifi = "==2020.6.20"
+chardet = "==3.0.4"
+click = "==7.1.2"
+idna = "==2.10"
+itsdangerous = "==0.24"
+numpy = "==1.19.1"
+pandas = "==1.1.0"
+python-dateutil = "==2.8.1"
+pytz = "==2020.1"
+requests = "==2.24.0"
+six = "==1.15.0"
+urllib3 = "==1.25.10"
+pytest = "==5.4.3"
+pytest-html = "==3.0.0"
+Flask = "==1.1.2"
+Flask-RESTful = "==0.3.8"
+Jinja2 = "==2.11.2"
+MarkupSafe = "==0.23"
+PyBluez = "==0.23"
+Werkzeug = "==1.0.1"
+opencv-python = "*"
+#opencv-contrib-python = "*"
+#opencv-contrib-python = "==4.1.0.25"
+#opencv-contrib-python-headless = "*"
```

Comment:
I have not been able to find something that works in a pipenv environment for this package.